pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/mercury/PostMercuryOrderRoute.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.dsm.route.mercury;
+
+import com.google.gson.Gson;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.dao.mercury.MercuryOrderDao;
+import org.broadinstitute.dsm.model.mercury.MercuryOrderRequest;
+import org.broadinstitute.dsm.pubsub.MercuryOrderPublisher;
+import org.broadinstitute.dsm.statics.RoutePath;
+import org.broadinstitute.dsm.statics.UserErrorMessages;
+import org.broadinstitute.lddp.handlers.util.Result;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+@Slf4j
+public class PostMercuryOrderRoute implements Route {
+    private static String projectId;
+    private static String topicId;
+    private static MercuryOrderPublisher mercuryOrderPublisher = new MercuryOrderPublisher(new MercuryOrderDao());
+
+    public PostMercuryOrderRoute(String projectId, String topicId) {
+        this.projectId = projectId;
+        this.topicId = topicId;
+    }
+
+    public static void publishMessage(MercuryOrderRequest mercuryOrderRequest, DDPInstance ddpInstance) {
```

Comment:
having method as `static` is not necessary here at all, removing static on the method will help to remove static on fields `projectId`, `topicId`, `mercuryOrderPublisher`.